web/editor_undo_bar.js

```diff
@@ -0,0 +1,101 @@
+/* Copyright 2024 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { AnnotationEditorType } from "../src/shared/util.js";
```

Comment:
Sorry, but you cannot import like this in the `web/`-folder since that could end up pulling in a very significant chunk of code in the *built* `viewer.mjs` file.

For an example of how this *should* be done, see [one example here](https://github.com/mozilla/pdf.js/blob/d37e4b08e4006b382b14b1596c5f70df7e93fbd9/web/pdf_viewer.js#L31-L39), however note that given the comment above this import shouldn't even be necessary here.